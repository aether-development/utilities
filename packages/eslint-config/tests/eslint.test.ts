// @ts-expect-error This file in the eyes of vitest is a module.
import eslintConfig from "../src";
import { describe, test, expect } from "vitest";

describe("ESLint Config", () => {
	test("should export rules", () => {
		expect(eslintConfig.root).toBe(true);
		expect(eslintConfig.parser).toBe("@typescript-eslint/parser");
		expect(eslintConfig).toMatchSnapshot();
	});
});
