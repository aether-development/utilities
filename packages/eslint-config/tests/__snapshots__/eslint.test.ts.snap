// Vitest Snapshot v1

exports[`ESLint Config > should export rules 1`] = `
{
  "env": {
    "browser": true,
    "commonjs": true,
    "es2017": true,
    "es2020": true,
    "es6": true,
    "jest": true,
    "node": true,
  },
  "extends": [
    "plugin:@typescript-eslint/recommended",
    "plugin:prettier/recommended",
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": 2020,
    "extraFileExtensions": [
      ".mjs",
    ],
    "project": "./tsconfig.eslint.json",
    "sourceType": "module",
  },
  "root": true,
}
`;
